src/redux/state.js

```diff
@@ -0,0 +1,27 @@
+const state = {
+  messagesData: [
+    { id: 1, name: 'Lorem, ipsum dolor sit amet  ' },
+    { id: 2, name: 'An praesentium tempore atque evenietdrey' },
+    { id: 3, name: 'architecto Placeat inventore officiis odit ipsa quibusdam' },
+    { id: 4, name: 'consectetur adipisicing elit. Officiis error' },
+    { id: 5, name: 'adipisicing elit. Officiis error' }],
+  dialogsData: [
+    { id: 1, name: 'Dymuch', src: 'https://upload.wikimedia.org/wikipedia/commons/9/9f/Dmitry_Medvedev_2016.jpg' },
+    { id: 2, name: 'Andrey', src: 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQ3ouc4dEwInAn-fzd0par_bEkPsHUxzx4J-nXg1sk-SC4ABqAt' },
+    { id: 3, name: 'Natalya', src: 'https://images11.cosmopolitan.ru/upload/img_cache/7b3/7b3b3da698f08bee6e9e7402053f6df5_ce_750x497x0x129_fitted_740x0.jpg' },
+    { id: 4, name: 'Igor', src: 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRNtwQUeVJ4vImAb6HiD2qFiEXchzJJKPDWUU96D33THFtBskw1' },
+    { id: 5, name: 'Adil', src: 'https://img.gazeta.ru/files3/784/11560784/skrskr2-pic905-895x505-4108.jpg' }],
+  friends: [
+    { id: 1, name: 'Dymuch', src: 'https://upload.wikimedia.org/wikipedia/commons/9/9f/Dmitry_Medvedev_2016.jpg' },
+    { id: 2, name: 'Natalya', src: 'https://images11.cosmopolitan.ru/upload/img_cache/7b3/7b3b3da698f08bee6e9e7402053f6df5_ce_750x497x0x129_fitted_740x0.jpg' },
+    { id: 3, name: 'Adil', src: 'https://img.gazeta.ru/files3/784/11560784/skrskr2-pic905-895x505-4108.jpg' }],
+  Path: {
+    dialog: 'dialog',
+  },
+  I18Y: {
```

Comment:
This does not belong to state in the current architecture.

Reply:
Sure